.github/workflows/ci.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         node: [8, 10, 12, 13]
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```

Comment:
Could I trouble you for a hammer PR on this one?  We shouldn't be rename/merging this 80 times :)

Reply:
this PR has already gone out to all repositories, and is the one Alex is working on landing as we speak.